src/react/types/types.ts

```diff
@@ -124,11 +124,18 @@ export interface SuspenseQueryHookOptions<
   | 'canonizeResults'
   | 'returnPartialData'
   | 'refetchWritePolicy'
-  | 'skip'
 > {
   fetchPolicy?: SuspenseQueryHookFetchPolicy;
   suspenseCache?: SuspenseCache;
   queryKey?: string | number | any[];
+
+  /**
+   * If `true`, the query is not executed. The default value is `false`.
+   *
+   * @deprecated We recommend using skipToken in place of the skip option. This 
+   * is more type-safe in most cases.
+   */
```

Comment:
```suggestion
   * @deprecated We recommend using `skipToken` in place of the skip option. This
   * is more type-safe in most cases.
   * @example recommended usage of `skipToken`:
   * ```js
   * import { skipToken, useSuspenseQuery } from '@apollo/client';
   *
   * const { data } = useSuspenseQuery(query, id ? { variables: { id } } : skipToken)
   * ```
   */
```

We have a teaching moment here and should use it :)

![image](https://github.com/apollographql/apollo-client/assets/4282439/9da2a1a3-7d28-41c9-8752-1faf44501023)
![image](https://github.com/apollographql/apollo-client/assets/4282439/d539855f-ff46-4ef8-886a-74f7ff02ae4d)

Reply:
PS: I'm not really happy with the "This is more type-safe in most cases." sentence - could we either skip that or change it to "This is more type-safe."?